packages/kit/src/runtime/server/utils.js

```diff
@@ -15,6 +15,7 @@ export function decode_params(params) {
 	for (const key in params) {
 		// input has already been decoded by decodeURI
 		// now handle the rest that decodeURIComponent would do
+		// TODO shouldn't we just use decodeURIComponent in the first place?
```

Comment:
answered this on Discord. no, unfortunately we can't. So I think we can probably remove this TODO

Reply:
removed